rocky/rocky/views/ooi_view.py

```diff
@@ -105,8 +105,10 @@ def get_queryset_params(self):
 
     def get_context_data(self, **kwargs):
         context = super().get_context_data(**kwargs)
-        context["observed_at"] = self.observed_at
+        context["observed_at"] = self.observed_at.strftime("%Y-%m-%d %H:%M")
+        context["observed_at_human_readable"] = self.observed_at.strftime("%B %d, %Y, %H:%M (%Z)")
```

Comment:
Since datetime formatting can be handled in the template, introducing two new variables for this looks redundant to me. It suggest applying formatting directly in the template instead